code/modules/lore_codex/news_data/main.dm

```diff
@@ -112,6 +112,8 @@
 		/datum/lore/codex/page/about_news,
 		)
 
+	var/newsindex = "103" //Update with number of latest article
```

Comment:
Surely this should be set on the fly, not hardcoded.

Reply:
Ideally yes, but considering news articles ALSO have to be hard-coded currently I didn't see any benefit to complex code like the Game Update News system uses.
If I ever work out/want to put in the excessive effort to add an easier way to upload news articles than going through the PR process, then that would absolutely be something that would change to match.

Reply:
That was my thought. Either grab some key from the news datum or just use the length of the list.